packages/api-server/src/index.ts

```diff
@@ -1,6 +1,41 @@
 #!/usr/bin/env node
 import yargs from 'yargs'
 
-import { apiCliOptions, apiServerHandler } from './cliHandlers'
+import {
+  apiCliOptions,
+  webCliOptions,
+  commonOptions,
+  apiServerHandler,
+  webServerHandler,
+  bothServerHandler,
+} from './cliHandlers'
 
-apiServerHandler(yargs.options(apiCliOptions).argv)
+const commandPath = yargs.argv.$0
+
+const positionalArgs = yargs.argv._
+
+// "bin": {
+//   "rw-api-server": "./dist/index.js",
+//   "rw-api-server-watch": "./dist/watch.js",
+//   "rw-log-formatter": "./dist/logFormatter/bin.js",
+//   "rw-server": "./dist/index.js"
+// },
+
+// suggest deprecating rw-api-server in favor of rw-server; TBD
```

Comment:
@thedavidprice I agree - I think we should depracate `rw-api-server` now. 

I think we'd need to change `rw dev` here: https://github.com/redwoodjs/redwood/blob/main/packages/cli/src/commands/dev.js - but no big deal. 

The reason I kept backwards compatibility before was because we were recommending using `rw-api-server` for serverful/self-hosted deployments in the past. I think now's the time to change it, if we're ever going to!

Reply:
Added a deprecation notice in `index.ts` and CHANGELOG. See commit https://github.com/redwoodjs/redwood/pull/4278/commits/773efd3d7be1e4bed063f343e461a22e59b22c95

Resolving this conversation. Please re-open as needed.